src/main/java/seedu/address/logic/commands/SetRsvpCommand.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.RsvpStatus;
+
+/**
+ * Set RSVP status for guests
+ */
+public class SetRsvpCommand extends Command {
+    public static final String COMMAND_WORD = "SetRsvp";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sets the RSVP status of a guest. "
+            + "Parameters: INDEX (must be a positive integer) "
+            + "STATUS (1 - Coming, 2 - Not Coming, 3 - Pending)\n";
+
+    public static final String MESSAGE_SET_SUCCESS = "RSVP status updated for guest: ";
+    public static final String MESSAGE_INVALID_INDEX = "ERROR: Please enter a valid index (from 1 to ";
+    public static final String MESSAGE_INVALID_ACTION = "ERROR: Please enter a valid action (1, 2, or 3)";
+
+    private final Index index;
+    private final int action;
```

Comment:
Should a new class be created for the action number (similar to the Index class)? The action numbers can be represented as an enum and I think it would adhere to separation of concerns more. It could potentially reduce the required changes to this class if more actions are added.